mkosi

```diff
@@ -2885,9 +2885,14 @@ def make_tar(args: CommandLineArguments,
     if for_cache:
         return None
 
+    if args.zstd:
+        filter_archive = '--zstd'
```

Comment:
I find the name `filter_archive` a bit curious. Why filter?

Reply:
I referred to tar's man:
```
-J, --xz
    filter the archive through xz 
```